notebooks/1. Generate reference data.ipynb

```diff
@@ -0,0 +1,217 @@
+{
```

Comment:
Wat vind je van deze naam en nr. van de notebook?

Reply:
Ja perfect. Is duidelijk denk ik.